js_modules/dagit/packages/core/src/runs/RunDetails.tsx

```diff
@@ -112,14 +113,38 @@ export const RunDetails: React.FC<{
   );
 };
 
+
 export const RunConfigDialog: React.FC<{run: RunFragment; isJob: boolean}> = ({run, isJob}) => {
   const [showDialog, setShowDialog] = React.useState(false);
   const {rootServerURI} = React.useContext(AppContext);
   const runConfigYaml = yaml.stringify(run.runConfig) || '';
+
+  const launchpadPath = () => {
+    const path = `/playground/setup-from-run/${run.id}`;
+
+    if (run.repositoryOrigin) {
+      return workspacePipelinePath({
+        repoName: run.repositoryOrigin.repositoryName,
+        repoLocation: run.repositoryOrigin.repositoryLocationName,
+        pipelineName: run.pipelineName,
+        isJob,
+        path,
+      });
+    }
+
+    return workspacePipelinePathGuessRepo(run.pipelineName, isJob, path);
+  };
+
   return (
     <div>
       <Group direction="row" spacing={8}>
-        <ButtonWIP icon={<IconWIP name="tag" />} onClick={() => setShowDialog(true)}>
+        <ButtonWIP 
+          icon={<IconWIP name="edit" />}
+          onClick={() => window.open(launchpadPath())}
+        >
+            Open in Launchpad...
+        </ButtonWIP>
```

Comment:
@braunjj Curious about your thoughts on how to add this to the Run page elegantly. I think we might run out of space in the header if this is added there, but I'm not sure where it should go.

Reply:
Can we do something like this where we have two primary action buttons, then an overflow menu that only contains the `download debug file` link for now.

![image](https://user-images.githubusercontent.com/2798333/161856167-99dc71aa-617e-48b3-ab81-b11098b9ac4e.png)

When Clicked...
![image](https://user-images.githubusercontent.com/2798333/161856098-d1511e13-c26b-4dcd-9759-067c375f04e2.png)